src/WebJobs.Script/OutOfProc/Http/HttpInvokerChannel.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Script.Description;
+using Microsoft.Azure.WebJobs.Script.Diagnostics;
+using Microsoft.Azure.WebJobs.Script.Eventing;
+using Microsoft.Azure.WebJobs.Script.Rpc;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Script.OutOfProc
+{
+    public class HttpInvokerChannel : IHttpInvokerChannel, IDisposable
+    {
+        private readonly string _rootScriptPath;
```

Comment:
`_rootScriptPath` and  `_eventManager` are unused

`_workerId` could likely become an auto property here.

Reply:
done!